package.json

```diff
@@ -16,11 +16,15 @@
     "start": "electron . --debug --log-debug --log-to-console --disable-http-cache --translation-watch",
     "pack": "npm run build && electron-builder --dir",
     "dist": "npx electron-builder --mac dmg --win nsis portable --linux AppImage deb",
-    "build": "node ./bin/build/build-info.js && npm run build-themes && npm run build-main && npm run builder",
-    "build-main": "tsc --build src/main",
-    "builder": "node ./bin/build/builder.js",
-    "builder-help": "npm run builder -- --help",
-    "build-scss": "npm run builder -- --styles",
+    "clean": "rm -rf ./tsc-dist ./html-dist ./dist",
+    "build": "npm run build:shared && npm run build:backend && npm run build:frontend",
+    "build:backend": "npm run build:backend:ts",
+    "build:backend:ts": "tsc --build src/main",
+    "build:shared": "node ./bin/build-shared-version-info.js",
```

Comment:
`build:shared` naming is a bit off here, don't ya think? it's more about stamping the build with build-info before building.

Reply:
I think it's fine. It's about building something for the shared source folder and it's used by backend & frontend. I think that's something related to shared.